### DISCUSSION TITLE
Package name improvement?

### DISCUSSION THREAD

**Discussion by kadyb** [r-lidar/lasR#76] [Ideas]
Currently there are 3 packages (`lidR`, `lasR`, `rlas`) with similar names that are not clear at first glance to new users. Maybe it would be worth considering emphasizing fast processing for `lasR`, if it is possible and it is not too late to make such change, like `fastlasR` or something?

**Comment by Jean-Romain** [comment]
Almost nobody knows `rlas`. It is exclusively used by `lidR` and should not be of interest to regular users. Maybe a comment on this point in the README would help.

`lidR` is `lidR`.

`lasR` is named similarly to `lidR` on purpose to emphasize that the two packages are related. I can hear that maybe it was not the best idea. Especially since `lasR` is a pure C++ program, and I plan to make it available also as standalone software and as a Python package or QGIS plugin, so the `R` is questionable. Branding is not my expertise.

I'm open to suggestions. What is your personal experience with that?

**Reply by kadyb** [reply]
> I plan to make it available also as standalone software and as a Python package or QGIS plugin

Oh, great! I didn't know you had such plans.

> I'm open to suggestions. What is your personal experience with that?

I would suggest bringing this issue to a wider discussion considering contributors and the community via social media. It may turn out that such a change will not be necessary or there will be some interesting suggestions. For me the differences between these packages are clear, but for new users they can be little confusing. I can write a post on Mastodon if you are interested.

**Reply by Jean-Romain** [reply]
Ok, I'm open to discussion. Feel free to contribute and discuss. I'm not saying I will change the name, but I can hear the community's arguments and change some things to keep everything clear and tidy.

Regarding this, you should know that a lot of things will change in the near future. `lidR` and `lasR` have been supported by Laval University, Québec, Canada since the beginning, using funding granted by my research programs as a PhD, Post-doc, and Research Associate. In September, I will resign from the university. To continue maintaining `lidR` and `lasR` as free and open-source tools, I will start offering consulting services, development, and training courses. The website of my conpagny is under construction, where the difference between `lasR` and `lidR` will be better explained. I plan to update the README files of `lasR`, `lidR`, `rlas`, and so on. I also plan to create a `README` for the `r-lidar` organization. Things are a bit messy right now, and I have a lot to deal with, but it will get tidier soon.

> Oh, great! I didn't know you had such plans.

Yes, but these are not short-term projects. I'm working alone, and it requires a lot of development, but the design of `lasR` has been made from the beginning to be independent of R itself. R is just an API, and other APIs could exist, such as a Python package, a standalone software, or a QGIS plugin. **"Could exist"** means I have nothing in the box right now but it is technically possible (this is not true for `lidR`).

Some experimental stuff for R is coming today. Stay tuned. If it works for R it will be recycled in other non-R projects.
